webapp-django/crashstats/crashstats/utils.py

```diff
@@ -54,6 +54,11 @@ def wrapper(request, *args, **kw):
         response = f(request, *args, **kw)
 
         if isinstance(response, http.HttpResponse):
+            if (
+                isinstance(response, http.HttpResponseBadRequest) and
+                not response['Content-Type'].startswith('application/json')
+            ):
+                response['Content-Type'] = 'application/json; charset=UTF-8'
```

Comment:
This seems weird. If this is a JSON view, then shouldn't it always return JSON?

At a bare minimum, this should get a comment explaining why you're picking this specific response out for special treatment and what the consequences of not doing it are.